app/controllers/social_networking/shared_item_controller.rb

```diff
@@ -49,29 +50,44 @@ def shared_item_params
       params.permit(:id, :participant_id, :page)
     end
 
-    def group_shared_items(participant)
-      SharedItem
-        .includes(:item, :comments, :likes)
-        .all
-        .select do |shared_item|
-        shared_item.item.try(:participant).try(:active_group).try(:id) ==
-          participant.active_group.id
-      end
+    def group_shared_items(participant, requested_page)
```

Comment:
I am just wondering if it would be beneficial to have a default here incase a page number isn't provided (or would that be unnecessary):
```
def group_shared_items(participant, requested_page = 1)
...
```